.changeset/polite-mugs-poke.md

```diff
@@ -0,0 +1,16 @@
+---
+"@quri/squiggle-components": patch
+"@quri/prettier-plugin-squiggle": patch
+---
+
+Autocompletion improvements:
+- suggest local function names
+- suggest parameter names
+- don't suggest unreachable vars (declared below or in unreachable local scopes)
+- different icon for local functions
```

Comment:
Where is this icon? It's not clear to me, very likely I just missed it.

Reply:
It's provided by Codemirror, based on `const type = node.type.is("FunctionName") ? "function" : "variable"` condition.

Here's how it looks:

<img width="310" alt="Captura de pantalla 2023-08-22 a la(s) 12 14 14" src="https://github.com/quantified-uncertainty/squiggle/assets/89368/900bbe3f-f86d-4c1a-a381-46bbbc6fc31f">

Later we could extend it and use custom icons, but Codemirror supports a few out of the box.